src/components/ui/Form.tsx

```diff
@@ -50,6 +53,10 @@ export default function Form({ onSubmit, onError }: FormProps) {
     onSubmit(parsed);
   };
 
+  const handleX = () => {
+    setTouched(true);
+  };
```

Comment:
[nits]
handleX だと命名的には色々なことができそうなので、アイコンの操作であることが明示的な命名の方がいいように思います〜

```suggestion
  const handleXIcon = () => {
    setTouched(true);
  };
```